src/main/java/org/commcare/formplayer/services/MenuSessionRunnerService.java

```diff
@@ -483,7 +482,6 @@ private void answerQueryPrompts(FormplayerQueryScreen screen,
         if (queryDictionary != null) {
             screen.answerPrompts(queryDictionary);
         }
-        screen.refreshItemSetChoices();
```

Comment:
can the same call be removed above on line 433 in `handleQueryScreen`?

Reply:
I think that sets the initial itemset choices on the dropdowns based on any defaults (there are test failures after removing that line). I have not looked into it super deeply though.